runtime/lua/vim/_editor.lua

```diff
@@ -810,6 +810,44 @@ function vim.deprecate(name, alternative, version, plugin, backtrace)
   end
 end
 
+--- Create builtin mappings (incl. menus).
+--- Called once on startup.
+function vim._init_default_mappings()
+  -- mappings
+
+  --@private
+  local function map(mode, lhs, rhs)
+    vim.api.nvim_set_keymap(mode, lhs, rhs, { noremap = true, desc = 'Nvim builtin' })
+  end
+
+  map('n',  'Y',      'y$')
+  -- Use normal! <C-L> to prevent inserting raw <C-L> when using i_<C-O>
+  -- See https://github.com/neovim/neovim/issues/17473
```

Comment:
since we're shifting things, can take this opportunity to unverbose a bit:
```suggestion
  -- Use normal! <C-L> to prevent inserting raw <C-L> when using i_<C-O>. #17473
```